### ISSUE TITLE
Missing mixins folder is crashing the tool

### ISSUE THREAD

**Issue by andreyfel** [rajasegar/eax#1]
I've tried this tool on my app and it crashed with the following error:
```
.../app/mixins: No such file or directory                                        │
│ ember-cli        ~3.19.0                        ││                                                                                                    │
    at ChildProcess.exithandler (child_process.js:303:12)                                                                                               │
    at ChildProcess.emit (events.js:315:20)       ││                                                                                                    │
    at maybeClose (internal/child_process.js:1021:16)                                                                                                   │
    at Socket.<anonymous> (internal/child_process.js:443:11)                                                                                            │
    at Socket.emit (events.js:315:20)             ││                                                                                                    │
    at Pipe.<anonymous> (net.js:674:12)           ││                                       
```

**Comment by ctjhoa** [comment]
Same thing for `app/utils` folder

**Comment by rajasegar** [comment]
Looks like I have hard coded a list of folders in home page. Will address this asap.
Thanks folks for reporting the same.

**Comment by rajasegar** [comment]
Fixed in latest version `0.7.1` please check and let me know
https://github.com/rajasegar/eax/commit/1ab1de3c7e037520ed5d440e1827eab21a7d4079

**Comment by andreyfel** [comment]
Confirmed, it works now!
Thanks for the quick fix!
